HelloDI/src/HelloDI.Tests/SecureMessageWriterTests.cs

```diff
@@ -16,7 +16,8 @@ public class SecureMessageWriterTests
         [Fact]
         public void SutIsMessageWriter()
         {
-            Assert.IsAssignableFrom<IMessageWriter>(CreateSecureMessageWriter());
+            var secureMessageWriter = CreateSecureMessageWriter();
```

Comment:
We're only allowed to perform one action, on each line of code. Hence, we need two lines for this test.